packages/react-devtools-shell/src/devtools.js

```diff
@@ -17,8 +17,17 @@ const {contentDocument, contentWindow} = iframe;
 // Helps with positioning Overlay UI.
 contentWindow.__REACT_DEVTOOLS_TARGET_WINDOW__ = window;
 
+// Initialize the backend and frontend early so that we have the messages that we
+// need during initial render
 initializeBackend(contentWindow);
 
+// Activate the backend only once the DevTools frontend Store has been initialized.
+// Otherwise the Store may miss important initial tree op codes.
+// activateBackend(contentWindow);
```

Comment:
You can delete this commented out line?

Also the comment above it seems outdated now.